build.ts

```diff
@@ -40,20 +40,32 @@ async function main() {
 	];
 	await Promise.all(scripts);
 
+	if (isDev() && releaseTarget === 'safari') {
+		await new Promise((resolve) => {
```

Comment:
Why do we need to have this sleep?

Reply:
The dev build does not await the completion of the build before continuing, the sleep allows the initial build processes to finish up before creating the xcode project.

There is probably a better way here long term, but this did nicely for development on my end for the time being.

Reply:
Ahhh is that because of the watch mode? Yeah we should probably think of a better way longer term - but right now it'll do :)